shared/test/middleware/helpers/test_storage_apps.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../../test_helper'
+require_relative '../../../middleware/helpers/storage_apps'
+
+class StorageAppsTest < Minitest::Test
+  def setup
+    @user_storage_ids_table = PEGASUS_DB[:user_storage_ids]
+  end
+
+  def test_get_anonymous_age_restricted_app
+    signedout_storage_id = @user_storage_ids_table.insert(user_id: nil)
+    signedin_storage_id = @user_storage_ids_table.insert(user_id: 20)
+
+    # create an applab project
+    channel_id = StorageApps.new(signedout_storage_id).create({projectType: 'applab'}, ip: 123)
+    assert_raises StorageApps::NotFound do
+      StorageApps.new(signedin_storage_id).get(channel_id)
+    end
```

Comment:
would be helpful to add short a comment like `# other user cannot access project owned by signed out user`, so that we know the significant part is whether the owner is signed in, rather than whether the viewer is signed in